lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -86,6 +86,10 @@ public function get_items_permissions_check( $request ) {
 			return new WP_Error( 'rest_forbidden_context', __( 'Sorry, you cannot view this resource with edit context.' ), array( 'status' => rest_authorization_required_code() ) );
 		}
 
+		if ( in_array( $request['orderby'], array( 'email' ), true ) && ! current_user_can( 'list_users' ) ) {
```

Comment:
`in_array( $request['orderby'], array( 'email' ), true )`  What does that help tell us exactly? Couldn't any user who can't list users not order them, not necessarily just by email?  I am bit confused by the conditional.  Is it here because the other fields are publicly exposed, so if someone wanted to they could order by the other fields? If that's the intention, we would need to add in `registered_date` as well because that is not publicly queryable as well.  If the conditional is for some other reason please let me know, I am probably overlooking something obvious.